kubernetes_env/benchmark/benchmark.py

```diff
@@ -143,22 +169,35 @@ def start_benchmark(fortio, filter_dirs, platform, threads, qps, time):
                       " Make sure you give the right path")
             return util.EXIT_FAILURE
 
-        log.info("Running fortio...")
-        # Might break for filter_dir/
-        fname = fd.split("/")[-1]
+        # wait for kubernetes set up to finish
+        time.sleep(10)
+        fname = Path(fd).name
```

Comment:
This conversion looks redundant?

Reply:
Since we would need to extract filename out and append it to our filepath later on to be used, I was thinking in case user can input something such as `path/to/filter/` or `path/to/filter/`